websocket/connection/wsConnClient_test.go

```diff
@@ -161,3 +161,18 @@ func TestWsConnClient_ReOpenAlreadyOpenedConnectionShouldError(t *testing.T) {
 
 	_ = conClient.Close()
 }
+
+func TestWsConnClient_IsOpen(t *testing.T) {
+	testServer := testscommon.NewHttpTestEchoHandler()
+	defer testServer.Close()
```

Comment:
Doesn't this `Close` return an error and this defer causes a waning on the IDE (smth like unhandled error)?